daal4py/bench.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2017-2019 Intel Corporation
+# Copyright (C) 2017-2020 Intel Corporation
 #
```

Comment:
Are these copy-paste files in three benchmarks: cuml, daal4py, sklearn? What for? Can't put it in one?

Reply:
Originally, this set of benchmarks only had sklearn and native implementations, so this wasn't a problem. But now, we might want to change the directory structure to be proper Python module stuff and access benchmarks using something like `python -m sklearn_bench.sklearn.ridge [ARGS...]`. Then we can use relative imports for this common file. The top-level runner and dataset creation file could also be included in the top-level module.
Switching to using this format isn't a big deal, but can be done in another PR.